### PR TITLE
Added the 'Quick Installation' section

### PR DESCRIPTION
**Pull request by schaubl**
Connected to #44.

Live demo: http://sixsq.lionel.pro:4000/documentation/administrator_guide/quick_installation.html

### CODE REVIEW

File: documentation/administrator_guide/quick_installation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- Disable SElinux
+- Configure nginx, HSQLDB and SlipStream
+- Replace the firewall configuration with the following:
+  - Allow all outgoing packets
+  - Allow loopback connections
+  - Allow incoming ICMP requests
+  - Allow incoming SSH connections (port 22)
+  - Allow incoming HTTP connections (port 80)
+  - Allow incoming HTTPS connections (port 443)
+  - Deny all others incoming connections
```
Comment (loomis) [inline documentation/administrator_guide/quick_installation.md]: "all others incoming..." -> "all other incoming..."

File: documentation/administrator_guide/index.md [hunk 2]
```diff
@@ -85,3 +85,18 @@ The following table shows the availability of the cloud connectors.
 |Physical/Fixed        |  open source | available    |
 |StratusLab            |  open source | available    |
 |VMWare vCloud         |  proprietary | available    |
+
+## SlipStream editions
```
Comment (loomis) [inline documentation/administrator_guide/index.md]: "editions" -> "Edition"

File: documentation/administrator_guide/index.md [hunk 3]
```diff
@@ -85,3 +85,18 @@ The following table shows the availability of the cloud connectors.
...
+
+SlipStream is available in two versions, the **Community** one and the
```
Comment (loomis) [inline documentation/administrator_guide/index.md]: "versions" -> "editions"

File: documentation/administrator_guide/index.md [hunk 4]
```diff
@@ -85,3 +85,18 @@ The following table shows the availability of the cloud connectors.
...
+**Enterprise** one.
+
+Proprietary connectors (see above) are only available on the Enterprise edition.
+
+For each version we maintain three repositories:
```
Comment (loomis) [inline documentation/administrator_guide/index.md]: "version" -> "edition"

File: documentation/administrator_guide/index.md [hunk 5]
```diff
@@ -85,3 +85,18 @@ The following table shows the availability of the cloud connectors.
...
+
+- Snapshots (most recent code base but may be unstable)
+- Candidates (should be stable but this is not for production)
```
Comment (loomis) [inline documentation/administrator_guide/index.md]: Would state this as "possibly stable releases not yet validated for production".

File: documentation/administrator_guide/installation.md [hunk 6]
```diff
@@ -71,9 +71,6 @@ After this start the database:
 
 The database files will be stored in `/opt/slipstream/SlipStreamDB/`.
 
-**NOTE: The startup script will indicate that the start failed, but
-the database is indeed running correctly.**
-
```
Comment (loomis) [inline documentation/administrator_guide/installation.md]: The startup script has been corrected and doesn't "fail" anymore?

File: documentation/administrator_guide/quick_installation.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  - Deny forwarding
+- Start nginx, HSQLDB and SlipStream
+
+These instructions assume that you will be using the prebuilt binary
+packages for SlipStream.  If you want to build your own packages from
+the source, refer to the Developers Guide.
```
Comment (loomis) [inline documentation/administrator_guide/quick_installation.md]: "Developers" -> "Developer"

File: documentation/administrator_guide/quick_installation.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+`candidate` or `release`
+
+    $ curl -sSfL https://raw.githubusercontent.com/slipstream/SlipStream/master/install/slipstream-install.sh | bash -s {edition} {repokind}
+
+Once the installation script is terminated a SlipStream server will be
+running but without any connectors.
+
+## Cloud Connectors
+
+The quick install script install all open-source connectors.
```
Comment (loomis) [inline documentation/administrator_guide/quick_installation.md]: "install" -> "installs"

File: documentation/administrator_guide/quick_installation.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+If you have installed the enterprise version, you have to install all
```
Comment (loomis) [inline documentation/administrator_guide/quick_installation.md]: "version" -> "edition"

File: documentation/administrator_guide/installation.md (hunk 6 above)
Comment (schaubl) [inline documentation/administrator_guide/installation.md]: Yes :v:

File: documentation/administrator_guide/quick_installation.md (hunk 7 above)
Comment (schaubl) [inline documentation/administrator_guide/quick_installation.md]: I copied this to an other place, so I will find it and fix it on the initial location as well.

**Pull request merged**